cloudbuild-publish.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - name: 'gcr.io/cloud-builders/gcloud'
+    script: |
+      #!/usr/bin/env bash
+      set -e
+      REPO_TOKEN="$(gcloud auth print-access-token)"
```

Comment:
Why can't this secret be handled the same way as `NPM_TOKEN`, and be stored in secret manager?

Reply:
This generates fresh token every time with service account because a normal token expires after 24 hours. 
NPM_TOKEN is non expiry token we get from npm account.

Reply:
Fair enough!